### DISCUSSION TITLE
M600 doesn't work 1.3.2?

### DISCUSSION THREAD

**Discussion by duraluminium** [Jyers/Marlin#963] [General Discussion]
I just tried running a filament change (extensions-post processing) from within Cura (4.9.0) and experienced a bug. (I am running ver. 1.3.2 E3V2-BLTouch-5x5-v4.2.2; Ender 3 V2) This inserts the following g-code line:

`M600 E0.00 L0.00 X0.00 Y0.00 ; Generated by FilamentChange plugin`

So no retraction whatsoever, manual reload. I also had the load/reload distance in the settings set to 0. 

What I experienced is that when the filament change occurred, the hotend moves to 0,0, beeps, I make the filament change making sure it goes all the way in (and oozes out) then hit confirm. The printer then purges a good amount of filament, which I pull off the tip. Then it goes to resume printing, but at this moment, it does a retract on the scale of 20-50mm, then immediately a ~10mm feed, so that the filament that purged fine is now quite a ways outside of the hotend. It then starts printing, of course with no filament available to feed. After a while of printing air, the filament eventually reenters the hotend and starts to print, but the print is ruined as a large chunk has been left 'blank'.

I tried modifying the different retract parameters but no joy. Every time this 'rogue' overdone retract occurred resulting in no filament being at the tip even after a successful purge.

I just updated to version 1.3.3 and the issue seems to have been resolved! I changed NO other settings. After the purge, just before resuming printing, it still does a small retract, but follows that up immediately with an equal feed, so they cancel out and there is filament ready to print at the tip.

Just wanted to flag this issue if others have been experiencing the same thing, and if Jyers might want to correct it in the 1.3.2 release...

Otherwise keep up the good work! Cheers

**Comment by Jyers** [comment]
I would recommend searching discussions first when you notice something strange, this has been documented thoroughly:
#597, #714, #867

Also as far as "correcting the release", that's what releases are for. 1.3.3 fixes any issues found in 1.3.2 and so on and so forth. Updating an older release would go against the entire nature of releases as a whole.

Anyway glad you got it working, hope this clears some things up!

**Reply by duraluminium** [reply]
Hi Jake, Thanks for the quick response and sorry for the bother. I googled the issue quite a bit but for some reason (the nearly identically written lol) previous threads didn't show up. Next time I'll be sure to search the Github thoroughly. Thanks for all the work you do! Cheers
